data/pubspec.yaml

```diff
@@ -29,6 +29,13 @@ dependencies:
   # flutter_uploader
   flutter_uploader: ^1.2.0
 
+  # flutter_downloader
+  flutter_downloader: ^1.5.2
```

Comment:
you should add an `ADR` for this library.

Reply:
Sure after all stuff completed